handbook/demand/README.md

```diff
@@ -57,6 +57,38 @@ To propose an ad, or a change to an ad:
 5. Make revisions and re-attend the meeting as needed until the planned change is executed.
 7. Create a calendar reminder to check ad performance two weeks from the date changes were made.
 
+### Measure intent signals
+
+Things to go look at to see if (a) a contact [/account] needs to be created/edited (b) if an account is worth going after or warming up (c) an account is sales-ready to assign a lead/rep
+(in order of how worthwhile it is to spend time looking at the intent signal)
+
+1. Accounts currently assigned to reps (i.e. pipeline + stage0 + pre-pipeline IQMs)
+2. Accounts with trending psychological progression (as measured by fleetdm.com website signups (i.e. new contacts ± contacts that have increased their psystage to a certain point)
+3. Accounts that fleeties have suggested to go after in ABM maneuver sheet
+4. [MacAdmins Slack traffic](https://macadmins.slack.com/archives/C0214NELAE7/p1722561481530559) in the #fleet AND #osquery channels (channel joins, posts, reactions, thread replies, thread reactions)
+5. [LinkedIn page follows](https://www.linkedin.com/company/71111416/admin/analytics/followers/)
+6. [GitHub stars to fleetdm/fleet](https://github.com/fleetdm/fleet/stargazers) from non-fleeties
+
+### Warm-up actions
+
+Things to do to warm up an account and move it towards sales-readiness after discovering relevant intent signals.
+
+After an intent signal is discovered:
+
+1. Create the account in SalesForce if it doesn't already exist
+2. Fix and merge any duplicate accounts and contacts.
+3. Update the "Marketing stage" to "Research-ready".
+4. Fix and complete any incorrect, mistagged, or half-filled contacts already on the account. Then research all of the existing contacts and make sure they have "role", "buying situation", "linkedinUrl", "psychological stage", "intent signals" completely filled out and correct.
+5. Research the company, if necessary, viewing its LinkedIn company page to verify the relevant headcount.
+
+After an account is marked "[Research-ready](https://fleetdm.lightning.force.com/lightning/r/Report/00OUG000001LerV2AS/view?queryScope=userFolders)".
```

Comment:
Will review this rest after changes above are made. Why? Everything below might be a third responsibility. It's hard to say before I see it all together.